test/brain/model_test.cpp

```diff
@@ -1,139 +1,184 @@
+                                    brain::CommonWorkloadDefaults::ESTOP_DELTA);
+}
+
+TEST_F(ModelTests, DISABLED_TimeSeriesEnsembleTest) {
+  auto lr_model = std::make_shared<brain::TimeSeriesLinearReg>(
+      brain::LinearRegWorkloadDefaults::BPTT,
+      brain::CommonWorkloadDefaults::HORIZON,
+      brain::CommonWorkloadDefaults::INTERVAL);
+  auto kr_model = std::make_shared<brain::TimeSeriesKernelReg>(
+      brain::KernelRegWorkloadDefaults::BPTT,
+      brain::CommonWorkloadDefaults::HORIZON,
+      brain::CommonWorkloadDefaults::INTERVAL);
+  auto lstm_model = std::make_shared<brain::TimeSeriesLSTM>(
+      brain::LSTMWorkloadDefaults::NFEATS,
+      brain::LSTMWorkloadDefaults::NENCODED, brain::LSTMWorkloadDefaults::NHID,
+      brain::LSTMWorkloadDefaults::NLAYERS, brain::LSTMWorkloadDefaults::LR,
+      brain::LSTMWorkloadDefaults::DROPOUT_RATE,
+      brain::LSTMWorkloadDefaults::CLIP_NORM,
+      brain::LSTMWorkloadDefaults::BATCH_SIZE,
+      brain::LSTMWorkloadDefaults::BPTT, brain::CommonWorkloadDefaults::HORIZON,
+      brain::CommonWorkloadDefaults::INTERVAL,
+      brain::LSTMWorkloadDefaults::EPOCHS);
+
+  size_t LOG_INTERVAL = 20;
+  size_t NUM_SAMPLES = 1000;
+  size_t NUM_FEATS = 3;
+  float VAL_SPLIT = 0.5;
+  bool NORMALIZE = false;
+  float VAL_THESH = 0.06;
+
+  auto model =
+      std::unique_ptr<brain::TimeSeriesEnsemble>(new brain::TimeSeriesEnsemble(
+          {lr_model, kr_model, lstm_model}, {0.33f, 0.33f, 0.33},
+          brain::LSTMWorkloadDefaults::BATCH_SIZE));
+  TestingForecastUtil::WorkloadTest(*model, WorkloadType::SimpleSinusoidal,
+                                    LOG_INTERVAL, NUM_SAMPLES, NUM_FEATS,
+                                    VAL_SPLIT, NORMALIZE, VAL_THESH,
+                                    brain::CommonWorkloadDefaults::ESTOP_PATIENCE,
+                                    brain::CommonWorkloadDefaults::ESTOP_DELTA);
+}
+
+TEST_F(ModelTests, DISABLED_AugmentedNNUniformTest) {
+  auto model = std::unique_ptr<brain::AugmentedNN>(new brain::AugmentedNN(
+      brain::AugmentedNNWorkloadDefaults::COLUMN_NUM,
+      brain::AugmentedNNWorkloadDefaults::ORDER,
+      brain::AugmentedNNWorkloadDefaults::NEURON_NUM,
+      brain::AugmentedNNWorkloadDefaults::LR,
+      brain::AugmentedNNWorkloadDefaults::BATCH_SIZE,
+      brain::AugmentedNNWorkloadDefaults::EPOCHS));
+  EXPECT_TRUE(model->IsTFModel());
+  size_t LOG_INTERVAL = 20;
+  size_t NUM_SAMPLES = 10000;
+  float VAL_SPLIT = 0.5;
+  bool NORMALIZE = false;
+  float VAL_THESH = 0.05;
+
+  TestingAugmentedNNUtil::Test(*model, DistributionType::UniformDistribution,
+                               LOG_INTERVAL, NUM_SAMPLES,
+                               VAL_SPLIT, NORMALIZE, VAL_THESH,
+                               brain::CommonWorkloadDefaults::ESTOP_PATIENCE,
+                               brain::CommonWorkloadDefaults::ESTOP_DELTA);
+}
+
+TEST_F(ModelTests, DISABLED_AugmentedNNSkewedTest) {

```

Comment:
I know we probably cannot enable them in the CI, but have you tested these tests locally?

Reply:
Yes, I've tested them.